frontend/src/scenes/trends/personsModalLogic.tsx

```diff
@@ -85,7 +85,7 @@ export function parsePeopleParams(peopleParams: PeopleParamType, filters: Partia
         ]
     }
     if (action?.properties) {
-        params.properties = [...(params.properties || []), ...action.properties]
+        params.properties = { ...(params.properties || {}), ...action.properties }
```

Comment:
I think this can still end up in an unstable state. action.properties is a list of prop filters not a property group

Reply:
Yes, precisely. Still problematic for persons on stickyness with a global filter set. Everywhere else is okay for now. This was a small enough section though that I felt okay to merge, and not worry about this yet.